### PR TITLE
Only backs up sensible SSH settings.

### PR DESCRIPTION
**Pull request by ISNIT0**
https://github.com/lra/mackup/issues/336

### CODE REVIEW

**Comment by RickWong** [comment]
This should be a no-brainer. You don't mess with someone's privates unless they consent to it.

**Comment by lra** [comment]
@RickWong Any config synced with mackup is private, that's the essence of it and I made it to sync my ssh keys in the 1st place. The problem is that it's now used by other people =)

**Comment by RickWong** [comment]
Difference is that ssh keys are **keys** not config (.ssh/config). For some users syncing ssh keys can indeed be convenient. The question here is: Should mackup sync **keys** by opt-out (default) or by opt-in?

Personally I found it rather unpleasant to find that Dropbox* obtained my ssh key, just because I used `ls -lah` to view the backup dir. Only after that discovery did I find this issue on Github. Had to go replace my ssh keypairs everywhere.

* I trust Dropbox, but not this much.

File: mackup/applications/ssh.cfg [hunk 1]
```diff
@@ -2,4 +2,6 @@
 name = SSH
 
 [configuration_files]
-.ssh
+~/.ssh/config
+~/.ssh/known_hosts
+~/.ssh/authorized_keye
```
Comment (sqren) [inline mackup/applications/ssh.cfg]: It seems like this should be 
```
[configuration_files]
.ssh/config
.ssh/known_hosts
.ssh/authorized_keys
```

**Comment by lra** [comment]
Dupe of https://github.com/lra/mackup/pull/295

**Pull request closed without merging**